src/main/resources/Babylon_zh.properties

```diff
@@ -297,6 +299,9 @@ compatibility.mode.info = \u8BF7\u8003\u8651\u5C06\u60A8\u7684\u540C\u884C\u7CFB
 changeLanguage = \u6539\u53D8\u8BED\u8A00
 startChat = \u5F00\u59CB\u804A\u5929
 
-Keyboardlayout.msg1 = Assisted - \u7528\u6237\u7AEF\u5F53\u524D\u4F7F\u7528\u4E0D\u540C\u7684\u952E\u76D8\u5E03\u5C40\uFF1A%s
-Keyboardlayout.msg2 = \u8FD9\u53EF\u80FD\u4F1A\u5BFC\u81F4\u95EE\u9898\uFF0C\u5C24\u5176\u662F\u5728\u8F93\u5165\u7279\u6B8A\u5B57\u7B26\u65F6\u3002
-Keyboardlayout.msg3 = \u5982\u679C\u60A8\u9047\u5230\u95EE\u9898\uFF0C\u8BF7\u786E\u4FDD\u60A8\u5728\u4E24\u53F0\u8BA1\u7B97\u673A\u4E0A\u4F7F\u7528\u76F8\u540C\u7684\u8BBE\u7F6E\u3002
\ No newline at end of file
+keyboardlayout.msg1 = Assisted - \u7528\u6237\u7AEF\u5F53\u524D\u4F7F\u7528\u4E0D\u540C\u7684\u952E\u76D8\u5E03\u5C40\uFF1A%s
+keyboardlayout.msg2 = \u8FD9\u53EF\u80FD\u4F1A\u5BFC\u81F4\u95EE\u9898\uFF0C\u5C24\u5176\u662F\u5728\u8F93\u5165\u7279\u6B8A\u5B57\u7B26\u65F6\u3002
+keyboardlayout.msg3 = \u5982\u679C\u60A8\u9047\u5230\u95EE\u9898\uFF0C\u8BF7\u786E\u4FDD\u60A8\u5728\u4E24\u53F0\u8BA1\u7B97\u673A\u4E0A\u4F7F\u7528\u76F8\u540C\u7684\u8BBE\u7F6E\u3002
+
+outdated.msg1 = \u529F\u80FD\u51CF\u5C11\u3002
```

Comment:
@rmshadows  may I ask you to have a brief look at the Chinese grammar please?

    zh_settings.html
    Babylon_zh.properties

Very appreciated, many thanks in advance!